config/resource-manager.hcl

```
@@ -384,7 +384,7 @@ service "networkfunction" {
 }
 service "newrelic" {
   name      = "NewRelic"
-  available = ["2022-07-01"]
+  available = ["2022-07-01", "2024-01-01"]
```

This API version isn't available yet:

```
$ az rest --method get --uri "https://management.azure.com/providers/NewRelic.Observability/operations?api-version=2024-01-01"
Not Found({"error":{"code":"InvalidResourceType","message":"The resource type 'operations' could not be found in the namespace 'NewRelic.Observability' for api version '2024-01-01'. The supported api-versions are '2022-07-01,2022-07-01-preview,2023-10-01-preview,2023-11-01-preview'."}})
```

```suggestion
  available = ["2022-07-01"]
  ignore    = ["2024-01-01"]
```